vignettes/articles/theme.Rmd

```diff
@@ -43,12 +55,29 @@ output:
 
 ![](images/theme-dark.png){.snippet}
 
-As shown above, a major benefit of theming via `{bslib}` is that theming of things like `gauge()`, `valueBox()`, and even `DT::datatable()` "just work" (that is, their styling defaults are informed by the `{bslib}` theme). In fact, these components even work with `{bslib}`'s real-time theming widget, which you can add to `flex_dashboard` by adding `runtime: shiny` to the documents yaml matter and calling `bslib::bs_themer()` (you can also do this with `runtime: shinyrmd`, but make sure `bslib::bs_themer()` is called in a code chunk with `context='server'`):
+<!--I'd love to see the fonts part separated with an L2 header so it is easier to skim and see, plus linkable-->
+
+As shown above, a major benefit of theming via `{bslib}` is that theming of things like `gauge()`, `valueBox()`, and even `DT::datatable()` "just work" (that is, their styling defaults are informed by the `{bslib}` theme). 
+
+These components work with `{bslib}`'s real-time theming widget, which you can add to `flex_dashboard` by adding either:
+
++ `runtime: shiny` to the document's yaml matter and calling `bslib::bs_themer()`, or
+
+<!--where should the bs_themer call happen? does it matter?-->
```

Comment:
It doesn't really matter, but I've been providing something like this as a template

````
---
runtime:shiny
output:
  flexdashboard::flex_dashboard:
    theme:
       version: 4
---

```{r setup, include = FALSE}
bslib::bs_theme()
```
````